scenes/tower-control.gd

```diff
@@ -0,0 +1,52 @@
+tool
```

Comment:
Does it need to be active in the editor? I think we can remove the `tool` keyword here and also in `radial-control.gd`. I don't remember why we put the tool mode there.

Reply:
Yeah, I also think it is useless, I'm gonna delete it straight away.